### ISSUE TITLE
App crash on Android when no properties passed to logCustomEvent

### ISSUE THREAD

**Issue by antpuleo2586** [Appboy/appboy-react-sdk#59]
Hi

I've noticed that if you pass no event properties to the `logCustomEvent` call, on Android the app crashes, whereas on iOS it's successful.

The error is:

```
2019-06-18 10:38:22.780 18481-18630/com.myapp E/unknown:ReactNative: Exception in native call
    java.lang.NullPointerException: Attempt to invoke interface method 'com.facebook.react.bridge.ReadableMapKeySetIterator com.facebook.react.bridge.ReadableMap.keySetIterator()' on a null object reference
        at com.appboy.reactbridge.AppboyReactBridge.populateEventPropertiesFromReadableMap(AppboyReactBridge.java:109)
        at com.appboy.reactbridge.AppboyReactBridge.logCustomEvent(AppboyReactBridge.java:104)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:158)
        at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
        at android.os.Handler.handleCallback(Handler.java:874)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
        at android.os.Looper.loop(Looper.java:198)
        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:232)
        at java.lang.Thread.run(Thread.java:764)

```

and it's happening here:
https://github.com/Appboy/appboy-react-sdk/blob/master/android/src/main/java/com/appboy/reactbridge/AppboyReactBridge.java#L109

So are these properties actually required, because the documentation suggests otherwise?
It seems redundant making the call like this: `logCustomEvent("event", {})`.

Cheers

Ant

**Comment by simonso-nzme** [comment]
I can confirm this is the case, it is crashing unless {} is added

**Comment by Bucimis** [comment]
Hi @antpuleo2586 and @simonso-nzme - this issue should be fixed in our latest release 1.14.0. Thanks for filing! :)
